tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/paths/missingOk.txt

```diff
@@ -28,3 +28,4 @@
 # More items ok to be missing
 //ldml/characters/parseLenients.* ; ok
 
+//ldml/layout/orientation/.* ; ok
```

Comment:
This was a bit of cleanup (should have been here from the start, since missingOk.txt is for all the path-specific exceptions.

Reply:
Maybe related to https://unicode-org.atlassian.net/browse/CLDR-14827

Reply:
This particular one wasn't, since it had been previously hardcoded.